cmd/image-inspector.go

```diff
@@ -18,24 +18,26 @@ func main() {
 	flag.StringVar(&inspectorOptions.DstPath, "path", inspectorOptions.DstPath, "Destination path for the image files")
 	flag.StringVar(&inspectorOptions.Serve, "serve", inspectorOptions.Serve, "Host and port where to serve the image with webdav")
 	flag.BoolVar(&inspectorOptions.Chroot, "chroot", inspectorOptions.Chroot, "Change root when serving the image with webdav")
+	flag.BoolVar(&inspectorOptions.Local, "local", inspectorOptions.Local, "If set the image wont be pulled but instead the -path will be used")
```

Comment:
this option means don't pull images trough docker and don't extract it (so do a local scan)... `scan-local` ?